lib/core/utils/validation.dart

```diff
@@ -14,25 +15,28 @@ String? validateEmail(
   if (!isValidEmail) {
     return S.of(context).enterYourEmail;
   }
+
   return null;
 }
 
 /// Validates password strength and complexity requirements
 String? validatePassword(
   String? input,
-  BuildContext context,
-) {
+  BuildContext context, {
+  int minNumberPassword = 8,
```

Comment:
why would you make it function param? we will not pass it here. Make it static const of validation class or just in file